ocs_ci/ocs/scale_lib.py

```diff
@@ -649,3 +651,49 @@ def check_and_add_enough_worker(worker_count):
             raise UnavailableResourceException(
                 "There is no enough worker nodes to continue app pod scaling"
             )
+
+
+def increase_pods_per_worker_node_count(count=500):
+    """
+    Function to increase pods per node count, default OCP supports 250 pods per node,
+    from OCP 4.6 limit is going to be 500, but using this function can override this param
+    to create more pods per worker nodes.
+    more detail: https://docs.openshift.com/container-platform/4.5/nodes/nodes/nodes-nodes-managing-max-pods.html
+    WARN: This function will perform Unscheduling of workers and back to Scheduled state
+    Please aware if there is any non-dc pods then expected to be terminated.
+
+    Args:
+        count (int): Required pods per node count
+
+    """
+    max_pods_template = templating.load_yaml(constants.PODS_PER_NODE_COUNT_YAML)
+    max_pods_template['spec']['kubeletConfig']['podsPerCore'] = 65
```

Comment:
just shouldn't `podsPerCore` be dynamic based on the hardware used?

Reply:
I don't think so but it depends on what kind of pod we are planning to deploy, eg: if it's nginx(without IO) then we can create 60+. Added value 65 based on manual testing where created 500 pods with 16 CPU and resources are half consumed, its a paper value to reach this limit need to use good hardware and pod.
Once any one of the limit is hit, either PodsPerCore or PodsPerNode we can't create more pods. Just want to ensure PodsPerNode count is slightly higher here, so this limit is not reached.

Reply:
@keesturam @Avilir Added param for PodsPerCore in function, resolving this comment.